users/views.py

```diff
@@ -154,6 +155,15 @@ def user_create_confirm(request, token):
         user.confirm_token = ''
         user.is_active = True
         user.save()
+        user.circle_set.create(
+            name='Me',
+            is_editable=False
+        )
+        user.circle_set.create(
+            name='Public',
+            is_editable=False
+        )
```

Comment:
So this means, that for every users there will be 2 circles created? Can't we just make 2 circles that are 'public' and 'me' for everyone?